test/distributed/org/apache/cassandra/distributed/test/RepairErrorsTest.java

```diff
@@ -124,18 +128,57 @@ public void testRemoteSyncFailure() throws Exception
             result = cluster.get(1).nodetoolResult("repair", KEYSPACE);
             result.asserts().success();
 
-            // Make sure we've cleaned up sessions and parent sessions:
-            Integer parents = cluster.get(1).callOnInstance(() -> ActiveRepairService.instance.parentRepairSessionCount());
-            assertEquals(0, parents.intValue());
-            Integer sessions = cluster.get(1).callOnInstance(() -> ActiveRepairService.instance.sessionCount());
-            assertEquals(0, sessions.intValue());
+            assertNoActiveRepairSessions(cluster.get(1));
 
             cluster.forEach(i -> Assertions.assertThat(i.logs().grep("SomeRepairFailedException").getResult())
                                            .describedAs("node%d logged hidden exception org.apache.cassandra.repair.SomeRepairFailedException", i.config().num())
                                            .isEmpty());
         }
     }
 
+    @Test
+    public void testRemoteStreamFailure() throws Exception
+    {
+        try (Cluster cluster = init(Cluster.build(3)
+                                           .withConfig(config -> config.with(GOSSIP)
```

Comment:
nit: Do you mind adding a comment why NETWORK is not needed for a test that looks like it needs streaming for repair to work?  I think the answer is because it isn't necessary for the test as ByteBuddy intercepts.